index.d.ts

```diff
@@ -1187,14 +1187,17 @@ declare namespace Moleculer {
 		Notepack: Serializer
 	};
 
-	class Validator {
+	class BaseValidator {
 		constructor();
 		init(broker: ServiceBroker): void;
 		compile(schema: GenericObject): Function;
 		validate(params: GenericObject, schema: GenericObject): boolean;
 		middleware(): ((handler: ActionHandler, action: ActionSchema) => any);
+		convertSchemaToMoleculer(schema: any): GenericObject;
 	}
 
+	class Validator extends BaseValidator {} // deprecated
```

Comment:
This actually appears to be exported as the `FastestValidator` class constructor and not the `BaseValidator` class constructor (see index.js).

Reply:
Yeah, you are right. Thank, I'm fixing.

Reply:
I can't change it because there is no definition for Fastest Validator. Definition of Fastest same as BaseValidator.